### PR TITLE
Add Annotators: ABACO, REL, FALCON

### LINKED ISSUE #326: Add FALCON as NER/NED tool
https://labs.tib.eu/falcon/

### LINKED ISSUE #346: Investigate EL ER tool REL
Obviously works with GERBIL ?! https://github.com/informagi/REL

### LINKED ISSUE #357: Adding ABACO to the list of default annotators
Hello,

First of all, thanks for maintaining GERBIL. It is a really useful tool.

I was wondering whether you could add ABACO to the list of default annotators. It exposes a NIF web service at http://tec.citius.usc.es/gerbilendpoint/

Currently, it only supports the D2KB experiment. Link to the paper: https://doi.org/10.1016/j.knosys.2020.105967

Thank you in advance.

### PR DESCRIPTION
**Pull request by LukasBluebaum**
Closes #357: ABACO - D2KB
Closes #346: REL - OKETask1
Closes #326: FALCON - C2KB

### CODE REVIEW

File: src/main/properties/annotators.properties [hunk 1]
```diff
@@ -190,6 +206,14 @@ org.aksw.gerbil.annotators.definition.PBOH.cacheable=true
 org.aksw.gerbil.annotators.definition.PBOH.class=org.aksw.gerbil.annotator.impl.nif.NIFBasedAnnotatorWebservice
 org.aksw.gerbil.annotators.definition.PBOH.constructorArgs=${org.aksw.gerbil.annotators.PBOH.serviceUrl}
 
+### REL (Radboud Entity Linker)
+org.aksw.gerbil.annotator.REL.ServieURL=http://gem.cs.ru.nl/api
```
Comment (TortugaAttack) [inline src/main/properties/annotators.properties:210]: if https is available that would be nice to use. :)

File: src/main/properties/annotators.properties [hunk 2]
```diff
@@ -18,6 +18,14 @@ org.aksw.gerbil.annotators.definition.Aida.cacheable=true
 org.aksw.gerbil.annotators.definition.Aida.class=org.aksw.gerbil.annotator.impl.aida.AidaAnnotator
 org.aksw.gerbil.annotators.definition.Aida.constructorArgs=${org.aksw.gerbil.annotator.Aida.ServieURL}
 
+### ABACO
+org.aksw.gerbil.annotators.ABACO.serviceUrl=http://tec.citius.usc.es/gerbilendpoint/
```
Comment (TortugaAttack) [inline src/main/properties/annotators.properties:22]: if https is available that would be nice to use. :)

File: src/main/properties/annotators.properties (hunk 1 above)
Comment (TortugaAttack) [reply src/main/properties/annotators.properties:210]: sadly not available

File: src/main/properties/annotators.properties (hunk 2 above)
Comment (TortugaAttack) [reply src/main/properties/annotators.properties:22]: works with https.

Comment (LukasBluebaum) [reply src/main/properties/annotators.properties]: Changed it, thanks. :)

**Comment by TortugaAttack** [comment]
@MichaelRoeder If there is no objection from your side, I will merge it within the next hour

**Comment by LukasBluebaum** [comment]
Don't merge it yet please, just checked the repository of REL and apparently they added a new endpoint some time ago. Testing that one right now, it would also support https.

**Comment by LukasBluebaum** [comment]
I updated the service URL of REL to the new one, which also uses https.

**Comment by MichaelRoeder** [comment]
Sorry to bring this up again. :disappointed: 
@TortugaAttack could you please check the functionality of the annotators again

**Comment by MichaelRoeder** [comment]
ABACO:
```
2021-08-11 18:45:39,803 [pool-1-thread-1] ERROR [org.aksw.gerbil.annotator.decorator.ErrorCountingAnnotatorDecorator] - <Got an Exception from the annotator (ABACO)>
GerbilException: Response has the wrong status: HTTP/1.1 502 Bad Gateway (error type -106: Got an unexpected exception while running the experiment.)
	at org.aksw.gerbil.http.AbstractHttpRequestEmitter.sendRequest(AbstractHttpRequestEmitter.java:185)
	at org.aksw.gerbil.annotator.impl.nif.NIFBasedAnnotatorWebservice.request(NIFBasedAnnotatorWebservice.java:187)
	at org.aksw.gerbil.annotator.impl.nif.NIFBasedAnnotatorWebservice.performAnnotation(NIFBasedAnnotatorWebservice.java:150)
	at org.aksw.gerbil.annotator.impl.nif.NIFBasedAnnotatorWebservice.performD2KBTask(NIFBasedAnnotatorWebservice.java:114)
	at org.aksw.gerbil.annotator.decorator.TimeMeasuringAnnotatorDecorator.performD2KBTask(TimeMeasuringAnnotatorDecorator.java:305)
	at org.aksw.gerbil.annotator.decorator.TimeMeasuringAnnotatorDecorator$TimeMeasuringD2KBAnnotator.performD2KBTask(TimeMeasuringAnnotatorDecorator.java:116)
	at org.aksw.gerbil.annotator.decorator.ErrorCountingAnnotatorDecorator.performD2KBTask(ErrorCountingAnnotatorDecorator.java:383)
	at org.aksw.gerbil.annotator.decorator.ErrorCountingAnnotatorDecorator$ErrorCountingD2KBAnnotator.performD2KBTask(ErrorCountingAnnotatorDecorator.java:134)
	at org.aksw.gerbil.annotator.decorator.SingleInstanceSecuringAnnotatorDecorator.performD2KBTask(SingleInstanceSecuringAnnotatorDecorator.java:315)
	at org.aksw.gerbil.annotator.decorator.SingleInstanceSecuringAnnotatorDecorator$SingleInstanceSecuringD2KBAnnotator.performD2KBTask(SingleInstanceSecuringAnnotatorDecorator.java:178)
	at org.aksw.gerbil.execute.ExperimentTask.runExperiment(ExperimentTask.java:300)
	at org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:143)
	at org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by MichaelRoeder** [comment]
FALCON: Seems to be fine, although it seems to take some time to answer
REL: Works

**Comment by MichaelRoeder** [comment]
ABACO works again.

**Pull request merged**